holopy/scattering/theory/mie.py

```diff
@@ -144,10 +152,9 @@ def _raw_cross_sections(self, scatterer, medium_wavevec, medium_index, illum_pol
 
         where I_0 is the incident intensity.  See van de Hulst, p. 14.
         """
-        if isinstance(scatterer, Scatterers):
-            raise InvalidScatterer(scatterer,
-                                        "Use Multisphere to calculate " +
-                                        "radiometric quantities")
+        if not self._can_handle(scatterer):
```

Comment:
This is too general for the corresponding error message. It refers specifically to Multisphere, but that is only valid for `Spheres` scatterers, and not arbitrary scatterers that cannot be handled by Mie.

Reply:
Per our discussion, I've changed this to
```python
if isinstance(scatterer, Spheres):
```

Reply:
(in commit 2f67510)